docs/html/topics/vcs-support.md

```diff
@@ -45,6 +45,15 @@ When passing a commit hash, specifying a full hash is preferable to a partial
 hash because a full hash allows pip to operate more efficiently (e.g. by
 making fewer network calls).
 
+- To use ssh scheme, just replace `https://` to `ssh://git@`
+
+````{admonition} Example
+```{pip-cli}
+$ pip install "wheel @ git+ssh://git@github.com/pypa/wheel.git@main"
```

Comment:
Why not add this to the code snippet above where similar things are being grouped?

Reply:
The code snippet above is for general case, so I start a new section for examples.